src/main/kotlin/gdx/liftoff/data/libraries/unofficial/thirdPartyExtensions.kt

```diff
@@ -715,7 +715,7 @@ class RegExodus : ThirdPartyExtension() {
  * UI toolkit with extra widgets and a different theme style.
  * Check the vis-ui changelog for what vis-ui versions are compatible
  * with which libGDX versions; vis-ui 1.5.0 is the default and is
- * compatible with libGDX 1.10.0.
+ * compatible with libGDX 1.11.0.
```

Comment:
Are we sure vis-ui 1.5.0 is actually compatible with 1.11.0 ? Last time this required telling vis-ui to skip a version check.

Reply:
[https://github.com/kotcrab/vis-ui/blob/9706f77/ui/src/main/java/com/kotcrab/vis/ui/VisUI.java#L96-L103](https://github.com/kotcrab/vis-ui/blob/9706f7764749c14bec18797d06f66f76cf28261a/ui/src/main/java/com/kotcrab/vis/ui/VisUI.java#L96-L103) fires but the demo runs fine once `sourceCompatibility` is updated. Is vis-ui really this sensitive to version changes?